edgexfoundry/device-rfid-llrp-go
# refactor!: Change configuration and provision watcher file formats to YAML

BREAKING CHANGE: Configuration and Provision Watcher files now uses YAML format







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-rfid-llrp-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions
1. start up edgex with no-secty
2. `make build`
3. run `./device-rfid-llrp-inventory -cp`
4. verify device llrp  starts up with no error
5. stop device-rfid-llrp-inventory
6.  run `./device-rfid-llrp-inventory -cp -o`
7.  verify device llrp starts up with no error

## New Dependency Instructions (If applicable)
